examples/example_thermodynamic_beginner.py

```diff
@@ -0,0 +1,184 @@
+# BurnMan - a lower mantle toolkit
```

Comment:
Can we break this new example into a separate PR? I think it needs some commenting to tell what is going on.